pkg/kubectl/kubectl.go

```diff
@@ -30,6 +30,11 @@ import (
 
 const (
 	kubectlAnnotationPrefix = "kubectl.kubernetes.io/"
+	Warn                    = `
+WARN: By default only active pods are displayed, to display terminated
+pods as well set -a, --show-all option. It is done intentionally to let see
```

Comment:
Suggest `WARNING: By default only active pods are displayed. To display terminated pods, set the -a (--show-all) option.`